src/workerd/server/server.c++

```diff
@@ -1392,6 +1491,29 @@ public:
       KJ_LOG(ERROR, exception);
     }
 
+    // Removes actors from `actors` after 70 seconds of last access.
+    kj::Promise<void> cleanupLoop() {
+      constexpr auto EXPIRATION = 70 * kj::SECONDS;
+
+      while (true) {
+        auto now = timer.now();
+        actors.eraseAll([&](auto&, kj::Own<ActorContainer>& entry) {
+
+          // TODO(now): What do we do if the actor is still in memory, but the last access time
```

Comment:
Hmmmm, I'm not fully up to date on this code but can you attach some sort of guard to promises we return for working with the actor? I notice onActorBroken() below at least.

Reply:
Sorry, mind expanding a bit? You mean like a `try...catch`?

Reply:
Ugh, I've lost context because of the outdated commit. I suspect that I was trying to say that you could track if the actor was evicted by if anybody was waiting on `onActorBroken()`.